apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/SessionPool.SessionPoolKeyStatistics.cs

```diff
@@ -0,0 +1,126 @@
+﻿// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License").
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at 
+//
+// https://www.apache.org/licenses/LICENSE-2.0 
+//
+// Unless required by applicable law or agreed to in writing, software 
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and 
+// limitations under the License.
+
+using Google.Cloud.Spanner.Common.V1;
+
+namespace Google.Cloud.Spanner.V1;
+public partial class SessionPool
+{
+    /// <summary>
+    /// A snapshot of statistics for a <see cref="SessionPoolKey"/> within a <see cref="SessionPool"/>.
+    /// </summary>
+    public sealed class SessionPoolKeyStatistics
+    {
+        /// <summary>

```

Comment:
If we're exposing the key publicly, do we need these properties at all? Users can just get to them via the key.